packages/fx-core/src/plugins/resource/function/resources/message.ts

```diff
@@ -14,10 +19,11 @@ export class InfoMessages {
     )}'.`;
 
   public static readonly installFunctionExtensions: string = "Installing Azure Functions binding.";
-  public static readonly noChange: string =
-    "Nothing to deploy; no changes detected since last deployment.";
-  public static readonly skipDeployment: string =
-    "Skip deployment.";
+  public static readonly noChange: string = "No changes detected since last deployment.";
+  public static readonly skipDeployment: string = `Skip deployment for no changes detected. Remove ${path.join(
+    PathInfo.solutionFolderName,
+    PathInfo.funcDeploymentFolderName
+  )} folder to force deployment.`;
```

Comment:
maybe: `To fully redeploy the function project, please remove xxx and rerun the command`.

Reply:
Agree. Updated.